pkg/webhook/utils/service.go

```diff
@@ -18,12 +18,16 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-// MutateLBService mutates ServiceExternalTrafficPolicyTypeLocal of LoadBalancer type service
-func MutateLBService(new, old *corev1.Service) error {
+const (
+	AlicloudLoadBalancerSpecAnnotationKey = "service.beta.kubernetes.io/alibaba-cloud-loadbalancer-spec"
+	AlicloudLoadBalancerSmallSpec         = "slb.s1.small"
```

Comment:
Out of curiosity/without understanding much about the contents of this PR, but why is this hard-coded here/does it make sense to make it configurable somewhere?

Reply:
> Out of curiosity/without understanding much about the contents of this PR

It because now Alicloud needs the annotation `service.beta.kubernetes.io/alibaba-cloud-loadbalancer-spec` if we want to create `small` specification of loadbalancers, and this type loadbalancer is current default setting.

> but why is this hard-coded here/does it make sense to make it configurable somewhere?

Yes , I've already done some changes on the code, make it configuration from `configmap`.